cmd/vclusterctl/cmd/create.go

```diff
@@ -28,7 +28,9 @@ import (
 )
 
 var (
-	AllowedDistros = []string{"k3s", "k0s", "k8s"}
+	AllowedDistros           = []string{"k3s", "k0s", "k8s", "eks"}
+	errorMessageIPFamily     = "expected an IPv6 value as indicated by " // Dual-stack cluster with .spec.ipFamilies=["IPv6"]
+	errorMessageIPv4Disabled = "IPv4 is not configured on this cluster"  // IPv6 only cluster
```

Comment:
remove errorMessageIPFamily & errorMessageIPv4Disabled as they are not used in your code

Reply:
This is part of the create command, right? I don't believe I want to remove that from here because it's used in k3s.go

Reply:
@rothgar they are now used in a different location and not needed anymore so its safe to delete those here